src/home_robot_hw/home_robot_hw/env/stretch_pick_and_place_env.py

```diff
@@ -259,6 +259,9 @@ def _handle_joints_action(self, joints_action: Optional[np.ndarray]):
             current_joint_positions = self.robot.model.config_to_hab(
                 self.robot.get_joint_state()[0]
             )
+            old_tilt = joints_action[-1]
+            joints_action[-1] *= 5
+            print(f"[ENV] WARNING: hard-coded upscaling of tilt commands! FIX THIS! From {old_tilt} to {joints_action[-1]}")
```

Comment:
```suggestion
```

Reply:
Definitely we should remove this - especially if the new RL policies are working better